ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -111,6 +111,7 @@ export default class AlertTableRow extends React.Component {
         {` ${text} `}
         <Link
           to={`./alerts?id=${alertId}`}
+          target="_blank"
```

Comment:
Can you also add rel="noopener noreferrer"
```suggestion
          target="_blank"
          rel="noopener noreferrer"
```